tests/foreman/virtwho/ui/test_esx.py

```diff
@@ -54,6 +57,14 @@ def form_data():
 
 
 class TestVirtwhoConfigforEsx:
+    @pytest.mark.tier2
+    def setup_method(self):
+        """Set up a clean env for tests."""
+        virtwho_cleanup()
+        guest_name, guest_uuid = get_guest_info(settings.virtwho.esx.hypervisor_type)
+        if Host.list({'search': guest_name}):
```

Comment:
I would recommend you to use `target_sat.cli.Host` and the target_sat fixture should be used in the autouse fixture.

Reply:
@jyejare Thanks for your suggestion, have already update it. please review it. Furthermore, I found the PR status has been changed to "Draft", how can update it to "Open"?

Reply:
Look for button `Ready for Review` at the bottom of the page in PR.